src/nwm_routing/src/nwm_routing/__main__.py

```diff
@@ -896,8 +895,23 @@ def main_v03(argv):
     # The inputs below assume a very pedantic setup
     # with each run set explicitly defined, so...
     # TODO: Make this more flexible.
+
     run_sets = forcing_parameters.get("qlat_forcing_sets", False)
+    
+    if type(run_sets) == list:
+        pass
+    else:
+        run_sets = nhd_io.build_qlat_date_range(forcing_parameters)
```

Comment:
@BrianAvant-NOAA Here is the check that determines if you want to pass the qlats in using the old block format or the dynamic start->end date setup.